.github/workflows/prod.yml

```diff
@@ -14,28 +14,27 @@ jobs:
     permissions:
       contents: write
       packages: write
-      id-token: write    
+      id-token: write
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
       - name: Set upgrade variables
         run: |
-            elixir ./deploy/upgrade/handler.exs all_keys ./deploy/upgrade/prod.config        
+            elixir ./deploy/upgrade/handler.exs all_keys ./deploy/upgrade/prod.config
             echo "RELEASE_FROM=$(elixir ./deploy/upgrade/handler.exs upgrade_from ./deploy/upgrade/prod.config)" >> $GITHUB_ENV
             echo "RELEASE_TO=$(elixir ./deploy/upgrade/handler.exs upgrade_to ./deploy/upgrade/prod.config)" >> $GITHUB_ENV
             echo "NAME=$(elixir ./deploy/upgrade/handler.exs name ./deploy/upgrade/prod.config)" >> $GITHUB_ENV
       - name: Setup Elixir
-        run: |
-            . ~/.asdf/asdf.sh
-            echo 'elixir v1.14.5-otp-25' >> ~/.tool-versions
-            echo 'erlang 25.3.2.7' >> ~/.tool-versions
-            elixir -v
+        uses: erlef/setup-beam@v1
+        with:
+          otp-version: '25.3.2.7'
+          elixir-version: '1.14.5'
```

Comment:
Please return it to its original state. [That doesn't work with the customer runner](https://github.com/supabase/supavisor/pull/226/files) and such changes require a sensitive and careful check because they can break the current deployment process.